client/src/screens/NewChild/index.js

```diff
@@ -13,8 +11,7 @@ import { CREATE_NEW_CHILD } from './graphql';
 const NewChild = props => (
   <Mutation mutation={CREATE_NEW_CHILD}>
     {(createChild, mutationProps) => (
-      <Layout>
-        <NavBar isUserSignedIn />
+      <div>
```

Comment:
same here?? I remember having to wrap the Layout in an extra div for styling so this may be the case here too...